src/schedule/zephyr_dma_domain.c

```diff
@@ -56,6 +56,27 @@ struct zephyr_dma_domain_data {
 	 * associated channel was triggered.
 	 */
 	bool triggered;
+	/* used to keep track of how many pipeline tasks point
+	 * to the same data structure.
```

Comment:
is it the same as just "number of tasks, sharing the same data structure?"

Reply:
Yep.